Packs/MicrosoftExchangeOnline/Integrations/EWSO365/EWSO365.py

```diff
@@ -2048,7 +2051,7 @@ def parse_incident_from_item(item):     # pragma: no cover
         incident["details"] = item.body
     incident["name"] = item.subject
     labels.append({"type": "Email/subject", "value": item.subject})
-    incident["occurred"] = item.datetime_created.ewsformat()
+    incident["occurred"] = item.datetime_received.ewsformat()
```

Comment:
Just a question regarding this change, is this to change the occurred time from when an email was created to when an email was received? Is it this case for incident?

Reply:
Probably mirror the logic for `last_modification_time` for a `last_received_time` makes the most sense in that case